pkgs/tools/admin/lexicon/default.nix

```diff
@@ -1,44 +1,38 @@
 { lib
-, python3
+, buildPythonApplication
```

Comment:
Change not needed -> https://github.com/NixOS/nixpkgs/pull/163010/files#r824255580

Reply:
Done